### PR TITLE
Add constant metric for showing binary version

### PR DESCRIPTION
**Pull request by odg0318**
## Proposed changes

I implemented new prometheus metric including Klaytn binary version information. The metric is showed like below.
```shell
$ curl -s http://localhost:61001/metrics | grep build_info
# HELP klaytn_build_info A metric with a constant '1' value labeled by version
# TYPE klaytn_build_info gauge
klaytn_build_info{cpu_arch="arm64",go_version="go1.18",version="v1.8.4+d98cdc851a"} 1
```

With the above, `version` label can be used for monitoring the Klaytn binary version.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues
- https://github.com/klaytn/klaytn/issues/1387

## Further comments

The package internally used for metrics doesn't support labels of metrics. And so I couldn't consider other monitoring tools except for Prometheus. Please review my codes at the point of view.

### CODE REVIEW

File: metrics/utils/version.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func init() {
+	g := prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace:   metricNamespace,
+		Subsystem:   "",
+		Name:        "build_info",
+		Help:        "A metric with a constant '1' value labeled by version",
+		ConstLabels: prometheus.Labels{"version": params.Version},
```
Comment (odg0318) [inline metrics/utils/version.go:30]: `ConstLabels` is commonly used to include some metadata like version, build info and so on.
https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#Opts

File: metrics/utils/metrics.go [hunk 2]
```diff
@@ -63,7 +64,7 @@ func StartMetricCollectionAndExport(ctx *cli.Context) {
 		logger.Info("Enabling metrics collection")
 		if EnabledPrometheusExport {
 			logger.Info("Enabling Prometheus Exporter")
-			pClient := prometheusmetrics.NewPrometheusProvider(metrics.DefaultRegistry, "klaytn",
+			pClient := prometheusmetrics.NewPrometheusProvider(metrics.DefaultRegistry, metricNamespace,
```
Comment (odg0318) [inline metrics/utils/metrics.go:67]: `metricNamespace` is used in `version.go` file.

**Comment by odg0318** [comment]
Test failure is my fault in the code? I don't know well.
```
FAIL	github.com/klaytn/klaytn/storage/database	14.729s
```

**Comment by odg0318** [comment]
It looks like PR is ready to be merged now.

**Review by aidan-kwon — APPROVED** [review]

**Review by kjhman21 — APPROVED** [review]

**Pull request merged**